cli/azd/internal/telemetry/telemetry.go

```diff
@@ -18,13 +22,20 @@ import (
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/sdk/resource"
 	"go.opentelemetry.io/otel/sdk/trace"
-
 	semconv "go.opentelemetry.io/otel/semconv/v1.10.0"
 )
 
+const azdAppName = "azd"
+
+// the equivalent of AZURE_CORE_COLLECT_TELEMETRY
+const collectTelemetryEnvVar = "AZURE_DEV_COLLECT_TELEMETRY"
+
 const telemetryItemExtension = ".trn"
-const devInstrumentationKey = "d3b9c006-3680-4300-9862-35fce9ac66c7"
-const prodInstrumentationKey = ""
+const (
+	devInstrumentationKey  = "d3b9c006-3680-4300-9862-35fce9ac66c7"
+	prodInstrumentationKey = ""
```

Comment:
Anything to worry about that we are exposing these keys directly in the source?

Reply:
No, it's the unfortunate reality of client-side telemetry and is pretty much standard implementation. You have to expose some credentials to the client. The only authorized operation is to push new logs which is fine